test/models/developer_test.rb

```diff
@@ -71,6 +71,13 @@ class DeveloperTest < ActiveSupport::TestCase
     assert_not_nil developer.errors[:bio]
   end
 
+  test "invalid if hero is too long" do
+    @developer.hero = "This is hero text is entirely too long for this field and will be invalid"
+
+    refute @developer.valid?
+    assert_not_nil @developer.errors[:hero]
```

Comment:
may be assert the error message itself, so if it's looking too odd by some chance, we can customize it

Reply:
@vamsipavanmahesh updated: https://github.com/joemasilotti/railsdevs.io/pull/100/files#diff-744524efd81cc7981472e7b3ce18a95469a168d333dd97435f19371a9d20036cR78